src/Ilios/CliBundle/Command/SendChangeAlertsCommand.php

```diff
@@ -0,0 +1,223 @@
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $isDryRun = $input->getOption('dry-run');
+
+        $alerts = $this->alertManager->findAlertsBy(['dispatched' => false, 'tableName' => 'offering']);
+        if (! count($alerts)) {
+            $output->writeln("<info>No undispatched offering alerts found.</info>");
+            return;
+        }
+
+        $templateCache = [];
+
+        $sent = 0;
+        // email out change alerts
+        foreach ($alerts as $alert) {
+            $output->writeln("<info>Processing offering change alert {$alert->getId()}.</info>");
+
+            $offering = $this->offeringManager->findOfferingBy(['id' => $alert->getTableRowId()]);
+            if (! $offering) {
+                $output->writeln(
+                    "<warning>No offering with id {$alert->getTableRowId()},"
+                    . " unable to send change alert with id {$alert->getId()}.</warning>"
+                );
+                continue;
+            }
+            // do not send alerts for deleted stuff.
+            $deleted = $offering->getSession()->isDeleted()
+                || $offering->getSession()->getCourse()->isDeleted()
+                || ! $offering->getSession()->getCourse()->getSchool();
+            if ($deleted) {
+                // @todo print another warning here? [ST 2015/09/30]
+                continue;
+            }
+
+            // convoluted way of identifying alert recipients
+            $schools = $alert->getRecipients();
+            /** @var SchoolInterface $school */
+            $recipients = [];
+            foreach ($schools->toArray() as $school) {
+                $schoolRecipients = trim($school->getChangeAlertRecipients());
+                if ('' === $schoolRecipients) {
+                    continue;
+                }
+                $recipients = array_merge($recipients, array_map('trim', explode(',', strtolower($schoolRecipients))));
+            }
+            array_unique($recipients);
+            if (empty($recipients)) {
+                $output->writeln("<error>No alert recipients for offering change alert {$alert->getId()}.</error>");
+                continue;
+            }
+
+            // get change alert history from audit logs
+            $history = $this->auditLogManager->findAuditLogsBy([
+                'objectId' => $alert->getId(),
+                'objectClass' => 'alert',
+            ], [ 'createdAt' => 'asc' ]);
+            $history = array_filter($history, function (AuditLogInterface $auditLog) {
+                $user =  $auditLog->getUser();
+                return isset($user);
+            });
+
+            $subject = $offering->getSession()->getCourse()->getExternalId() . ' - '
+                . $offering->getStartDate()->format('m/d/Y');
+
+            $school = $offering->getSession()->getCourse()->getSchool();
```

Comment:
I feel like the template should be driven by the alert recipients as opposed to the primary school of the course.  I'm probably wrong though.